crash-recovery/README.md

```diff
@@ -0,0 +1,78 @@
+```
+shortname: ?/CRASH-RECOVERY
+name: Restore system state after crash
+type: standard
+status: raw
+editor: Vanshdeep Singh <vanshdeep@bigchaindb.com>
+```
+
+## Abstract
+Unexpect crashes are bound to occur when BigchainDB is deployed in production which could leave the system in an inconsistant state. Hence it is necessary that an appropriate methodology should be developed to restore the system to a consistant state.
```

Comment:
You could say a bit more here, even though it's an abstract. For example.

What two things can become inconsistent with each other, about what value?